test/unittests.jl

```diff
@@ -1697,3 +1698,8 @@ end
     y = Quantity(2.0im, mass=1)
     @test_throws DimensionError x^y
 end
+
+@testset "Register Units" begin
+    @register_unit MyV u"V"
+    @test MyV === u"V"
```

Comment:
Should test for `u"MyV"`

Also include a test for symbolic units, i.e., `us"MyV"`

Reply:
To get this working you might need to let `UNIT_SYMBOLS` be a regular vector rather than a tuple FYI.